package.json

```diff
@@ -43,22 +43,25 @@
     "postcss-preset-env": "^6.7.0",
     "postcss-safe-parser": "^4.0.1",
     "regenerator-runtime": "^0.13.3",
-    "sass-loader": "7.3.1",
-    "style-loader": "^1.0.0",
+    "sass-loader": "^8.0.0",
+    "style-loader": "^1.1.2",
     "terser-webpack-plugin": "^2.3.1",
-    "webpack": "^4.41.3",
+    "webpack": "^4.41.5",
     "webpack-assets-manifest": "^3.1.1",
     "webpack-cli": "^3.3.10",
     "webpack-sources": "^1.4.3"
   },
   "devDependencies": {
-    "eslint": "^6.6.0",
+    "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-import": "^2.19.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.16.0",
+    "eslint-plugin-react": "^7.17.0",
     "jest": "^24.9.0"
   },
+  "resolutions": {
+    "terser-webpack-plugin": "^2.3.1"
+  },
```

Comment:
> as a result, we have two terser-webpack-plugin ... I think terser-webpack-plugin will not be updated for webpack 4

It is ok that we have 2 versions of terser. Webpack needs to use the version of terser that it was built to support. There is no need to micromanage the lock file 👍.